Delectio/translator-backend
# Create route for translating text

#### What does this PR do?
Create route for translating text

#### Description of Task to be completed?
- Create versioned route and controller
- Implement translating text using the route

#### How should this be manually tested?
- Clone repository
- Start server using `npm start`
- On postman (or other API service), access the POST route `http://localhost:8080/translate` 
- Request body should include `text`, the text to be translated and `lang`, the language of the text (e.g `fr` for french or `ru` for russian
- Response should be returned, including the text in English

#### Any background context you want to provide?
- Text can only be translated to English
- Language auto-detection is not implemented

#### What are the relevant pivotal tracker stories?(if applicable)
NA

#### Screenshots
<img width="979" alt="Screenshot 2019-11-02 at 14 10 25" src="https://user-images.githubusercontent.com/42358974/68071431-8c1c1980-fd7a-11e9-8220-95321a47c987.png">
